navigator/backend/src/main/scala/com/digitalasset/navigator/config/Arguments.scala

```diff
@@ -154,6 +155,12 @@ object Arguments {
           }
         )
 
+      opt[Unit]("ignoreProjectParties")
```

Comment:
I suggest using kebab-case to be consistent with the other options:

```suggestion
      opt[Unit]("ignore-project-parties")
```